### PR TITLE
Expiring command & Expiry date field

### PR DESCRIPTION
**Pull request by Denise-Ng**

### CODE REVIEW

**Review by kevvvinn — DISMISSED** [review]
Thank you, Denise. Looks good!

File: src/main/java/csdev/couponstash/storage/JsonAdaptedCoupon.java [hunk 1]
```diff
@@ -25,20 +28,27 @@
...
      * Constructs a {@code JsonAdaptedCoupon} with the given coupon details.
      */
     @JsonCreator
+
     public JsonAdaptedCoupon(@JsonProperty("name") String name,
                              @JsonProperty("phone") String phone,
                              @JsonProperty("savings") JsonAdaptedSavings savings,
-                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged, 
+                             @JsonProperty("expiry date") String expiryDate,) {
```
Comment (kevvvinn) [inline src/main/java/csdev/couponstash/storage/JsonAdaptedCoupon.java]: hais my bad, one extra comma

File: src/main/java/csdev/couponstash/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -22,21 +23,35 @@
...
+          new Coupon(new Name("Irfan Ibrahim"), new Phone("92492021"),
                 new Savings(new MonetaryAmount(1d),
                         Arrays.asList(new Saveable("Brattby Bag"))),
-                getTagSet("classmates")),
+                getTagSet("classmates"), new ExpiryDate("1-10-2020")),
+          
+          
             new Coupon(new Name("Roy Balakrishnan"), new Phone("92624417"),
-                new Savings(new PercentageAmount(100d)), getTagSet("colleagues"))
+                new Savings(new PercentageAmount(100d)), getTagSet("colleagues")), new ExpiryDate("11-9-2020"))
```
Comment (kevvvinn) [inline src/main/java/csdev/couponstash/model/util/SampleDataUtil.java]: getTagSet("colleagues"),

**Review by ChesterSim — APPROVED** [review]
Looking great to me!

**Pull request merged**